packages/client.api/src/mapping/DataValueMapping.ts

```diff
@@ -54,4 +64,22 @@ export interface DataValueClientToWire {
   short: number;
   string: string;
   timestamp: string;
+  set: Set<any>;
+  twoDimensionalAggregation: {
```

Comment:
I'm still playing around with these types, was trying to get rid of the `unknown`s I initially had